app/state_machines/configurable_state_machine/machine.rb

```diff
@@ -128,6 +124,18 @@ def next_state_for_event(event, params)
                                     message: "Not permitted to trigger event")
       end
     end
+
+    def self.states
+      ['unassigned',
+       'awaiting_responder',
+       'drafting',
+       'pending_dacu_clearance',
+       'pending_press_office_clearance',
+       'pending_private_office_clearance',
+       'awaiting_dispatch',
+       'responded',
+       'closed']
+    end
```

Comment:
Minor thing: no need to disable cyclomatic complexity or method length for the`self.states` method. You can also move these declaration to the method definiton line (see https://stackoverflow.com/a/42214139):

```
def next_state_for_event(event, params) #rubocop:disable Metrics/CyclomaticComplexity Metrics/MethodComplexity
```

Reply:
Also, as this is a class method, how about moving it to the top of the class?